src/index.js

```diff
@@ -22,3 +24,24 @@ const server = new ApolloServer({
 server.listen({ port: process.env.PORT }).then(() => {
   console.log(`🚀 Server ready at ${process.env.PORT}`);
 });
+
+// Connect Sequelize to Postgres database
+const Sequelize = require("sequelize");
+const sequelize = new Sequelize("alexmerecka", "alexmerecka", "password", {
```

Comment:
sequelize lets you use a sql url string instead of having to split everything up. Also this theoretically is the username and password to your database. If this were production you'd want to hide this string and not check it in.

Reply:
Put URL into .env file and using process.env.DEV_DATABASE_URL now.